### DISCUSSION TITLE
adding circles in "Creating vector data interactively without coding"

### DISCUSSION THREAD

**Discussion by shi093** [opengeos/leafmap#198] [Q&A] [answered]
In Colab, when click the circle in the drawing tool, a point is created, instead of a circle. Thanks.

**Comment by giswqs** [comment, marked as answer]
This is an ipyleaflet bug https://github.com/jupyter-widgets/ipyleaflet/issues/876. There is not much I can do about it. You can either downgrade ipyleaflet to v0.13.3 or wait until ipyleaflet to fix the bug. 

```
pip install -U ipyleaflet==0.13.3
```

**Reply by shi093** [reply]
Good to know, thanks a lot.
